### ISSUE TITLE
Running sync server?

### ISSUE THREAD

**Issue by MoeMorox** [brave/sync#58]
Hi,

sorry for the inconvenience, this is more a question, maybe "half"-a-duplicate of #57.
Will you be able to run the sync server on your own machine without anything like AWS and use it on all clients (including Android)?

Thanks for your work btw!

**Comment by ayumi** [comment]
Yes, that would be sweet! I think it's important that users can self host and own their data.

We rely on S3 to put and query records, so for a first iteration something S3 compatible would be convenient. https://github.com/minio/minio looks like it might work.

**Comment by sebalinux** [comment]
I think this is a very important and useful feature, Mozilla Firefox allow this in a quite simple manner. I cannot think it is a technical problem for you, there are any other obstacles??

**Comment by diracdeltas** [comment]
The sync team has been busy getting sync out the door on all Brave platforms, so we haven't gotten around to this. However, I think it should be pretty straightforward:

1. Add a command line flag in https://github.com/brave/brave-browser to specify the URL for the sync credentials server (assuming you want to run that yourselves too) and the s3 bucket
2. Make sure the code in this repository reads that and uses it if set; otherwise it falls back to the Brave defaults

**Comment by avbfr** [comment]
Hello,
First, thanks for your work, Brave is a very good browser !

It's just missing this feature ;-)

Any news on this ? 

Regards.
